testcases/unit/pintest.c

```diff
@@ -22,11 +22,12 @@ int main(int argc, char *argv[])
 	switch(getopt(argc, argv, "pn")) {
 	case 'p':
 		pin = pin_prompt(&buf, "Enter the PIN: ");
-		printf("pin: %s\n", pin);
+		/* AIX prints an empty string instead of Linux-style (null) */
```

Comment:
maybe also remove the comment ? I guess the code is pretty much self explanatory :-)

Reply:
I would prefer to keep this in, as the `printf` line is no longer platform-dependent. It'd be better if the differences in platform behaviours are highlighted.